src/Form/Type/PhoneNumberType.php

```diff
@@ -61,12 +64,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
             $countryChoices = [];
 
-            foreach (Countries::getNames() as $region => $name) {
-                if (false === isset($countries[$region])) {
+            foreach ($countries as $regionCode => $countryCode) {
```

Comment:
Why this change? It's not equivalent to what was here before. Therefore it potentially breaks the compatibility.

Reply:
The results is almost the same. I made this change to keep the order given in the `countries` configuration entry.

Reply:
If I'm not mistaken, the performances are not the same this way because before the data was loaded and browsed only once in the buffer of the component while now we have it as many times as countries support x2 (exists & getName)

Reply:
You're right ! Not sure its a big deal. Let me check :)

Reply:
I would not be so convinced. I would like to have a verification on this before

Reply:
The performance overhead is actually small. But I'm surprised there's indeed a topic here. That said, a solution can be quickly found here:

```php
Countries::exists($regionCode)
```

could be changed easily with

```php
$intlCountries = Countries::getNames();
// and later
if (!isset($intlCountries[$regionCode])
```